airbyte-integrations/connectors/source-sentry/source_sentry/sentry.yaml

```diff
@@ -0,0 +1,107 @@
+definitions:
+  page_size: 50
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_sentry/schemas/{{ options.name }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: []
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    url_base: "https://{{ config.hostname }}/api/0/"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config.auth_token }}"
+  paginator:
+    type: LimitPaginator
+    url_base: "*ref(definitions.requester.url_base)"
```

Comment:
why are we passing url_base to the paginator?

Reply:
a bit of an implementation detail, but it's to replace the base url from the cursor when updating the path. although maybe that could be done in the retriever. left a [comment on this](https://github.com/airbytehq/airbyte/issues/15575#issuecomment-1212541353)